.github/workflows/close-stale-issues.yml

```
@@ -22,7 +22,7 @@ jobs:
     if: github.repository == 'nodejs/help'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/stale@v4
+      - uses: actions/stale@v8
```

OpenSSF and GitHub best practices would suggest we use a direct sha

```suggestion
      - uses: actions/stale@1160a2240286f5da8ec72b1c0816ce2481aabf84 # v8.0.0
```

---

True, but those “best practices” basically kill free-flowing bugfixes and security updates. I usually recommend only pinning to a sha when not doing so risks leakage of secrets.

---

are we good to keep it like that @bmuenzenmeyer ?